### ISSUE TITLE
Summary of Issues

### ISSUE THREAD

**Issue by CookiesNCream** [impact-learning/BJDW#1]
https://docs.google.com/spreadsheets/d/1qzFA4wXFOAWU8DBGWECgOLRsOJDCZ05W1ktj3MMe8Gg/edit

**Comment by skyuplam** [comment]
@CookiesNCream I don't understand why there are so many interactions to the map? Isn't it only visualizing the situation of the event on the big screen? I am expecting that the visualization is explicit and should minimize interaction to users. I remember that we are not going to implement any visualization on the mobile web app for the users, so what does `Press "To Impact Learning" to Navigate to Map` actually mean?

**Comment by gitnicos** [comment]
Yes, I agree. This creeped in somehow in Nill's document. All the visualization of analytics should be done in Desktop version as a priority not in mobile version. Obviously, since we are writing code in Web App mode, it should not matter, but I agree that fine tuning for glitch-less mobile visualization may take sizable time, which is not a high on priority.

**Comment by CookiesNCream** [comment]
I also think there are too many interactions to the map. Those interactions were brought up during the meeting and in Neill's document. I will remove that requested navigation to map user action and the hover over map user actions. The only user action on the map would be dragging the ruler across the timeline to reflect the dynamic updates on the map.
 

    On Wednesday, August 31, 2016 6:31 AM, Terrence Lam <notifications@github.com> wrote:
 

 @CookiesNCream I don't understand why there are so many interactions to the map? Isn't it only visualizing the situation of the event on the big screen? I am expecting that the visualization is explicit and should minimize interaction to users. I remember that we are not going to implement any visualization on the mobile web app for the users, so what does Press "To Impact Learning" to Navigate to Map actually mean?—

**Comment by gitnicos** [comment]
We still need to leave dimensionality choice between layers (1,2,3) through clicking on checkboxes. So to say "predefined queries/analytics" feel.

**Comment by CookiesNCream** [comment]
I have indicated the clicking on checkboxes for pre-defined queries for layers 1, 2, 3 as another user action on the map. 

    On Wednesday, August 31, 2016 11:33 AM, gitnicos <notifications@github.com> wrote:
 

 We still need to leave dimensionality choice between layers (1,2,3) through clicking on checkboxes. So to say "predefined queries/analytics" feel.—
